software/docs.yml

```
@@ -0,0 +1,12 @@
+name: "Docs"
+website_url: "https://docs.numerique.gouv.fr/"
+source_code_url: "https://github.com/suitenumerique/docs"
+description: "A collaborative note taking, wiki and documentation platform that scales."
+licenses:
+  - MIT
+platforms:
+  - Docker
```

The [install instructions](https://github.com/suitenumerique/docs/blob/main/docs/installation.md) seem to only mention K8S, where did you get docker from, did I miss something?
```suggestion
platforms:
  - K8S
```

---

Ah, I somehow didn't see that.